_posts/2017-09-11-DevSwap.markdown

```diff
@@ -0,0 +1,109 @@
+---
+layout: epic
+title: "DevSwap: An Experiment in Randomizing Teams"
```

Comment:
I might hyphenate it instead (dev-swap) for easier reading and googling. No need to invent new words.

Reply:
Idk, I kind of like DevSwap. By giving it a name it's easier to reference - kind of like Lunch and Learn. Maybe "Team Swap" is less obnoxious?